config/reports.json

```diff
@@ -8,51 +8,59 @@
 		"chrome-ux-report"
 	],
 	"_featured": [
-		"state-of-javascript"
+		"state-of-the-web"
 	],
 	"_metrics": {
 		"bytesCss": {
 			"name": "CSS Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external stylesheets requested by the page. An external stylesheets is identified as a resource with the `css` file extension or a MIME type containing `css`."
 		},
 		"bytesFont": {
 			"name": "Font Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all fonts requested by the page. A font is identified as a resource with the `eot`, `ttf`, `woff`, `woff2`, or `otf` file extensions or a MIME type containing `font`."
 		},
 		"bytesHtml": {
 			"name": "HTML Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all HTML documents requested by the page. An HTML document is identified as a resource with the `html` or `html` file extensions or a MIME type containing `html`."
 		},
 		"bytesImg": {
 			"name": "Image Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external images requested by the page. An external image is identified as a resource with the `png`, `gif`, `jpg`, `jpeg`, `webp`, `ico`, or `svg` file extensions or a MIME type containing `image`."
 		},
 		"bytesJs": {
 			"name": "JavaScript Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external scripts requested by the page. An external script is identified as a resource with the `js` or `json` file extensions or a MIME type containing `script` or `json`."
 		},
 		"bytesOther": {
 			"name": "Other Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all unidentified resources requested by the page. An unidentified resource is one that does not match a known script, font, image, stylesheet, XML file, video, HTML document, or plaintext file."
 		},
 		"bytesTotal": {
 			"name": "Total Kilobytes",
 			"type": "KB",
-			"description": "The sum of <a href=\"https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-transfersize\">transfer size</a> kilobytes of all resources requested by the page.",
+			"description": "The sum of [transfer size](https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-transfersize) kilobytes of all resources requested by the page.",
 			"timeseries": {
 				"redundant": true
 			}
 		},
 		"bytesVideo": {
 			"name": "Video Bytes",
 			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all videos requested by the page. A video is identified as a resource with the `mp4`, `swf`, `f4v`, or `flv` file extensions or a MIME type containing `flash`.",
```

Comment:
What about WebM, etc?

Meta question: where are these definitions coming from? Could we simplify all this by pointing to the source?

Reply:
> WebM

While going through the source code for these metrics, I definitely got the feeling that some were not up to date. I'm happy to take a TODO to modernize the metrics, while migrating from the old PHP/MySQL code to BigQuery.

All bytes/requests metrics are defined here: https://github.com/HTTPArchive/httparchive/blob/b52c778f67f7d3217e5c1e763372d4ee0a66e5a3/bulktest/batch_lib.inc#L933-L973

Reply:
Yikes, yeah.. that needs some work. Doesn't WPT already have this sort of logic somewhere? I'd be great if we could avoid having to reinvent the wheel. /cc @pmeenan

Reply:
Either way, I'm going to be rewriting all this categorization logic as we move off MySQL so I don't think we should block on it now. When we get there, I do agree that it's worth updating the definitions.

LMK if there's anything else in this PR that needs attention.

Reply:
sg. should we file a bug for this, so we can keep track?

Reply:
Filed https://github.com/HTTPArchive/beta.httparchive.org/issues/23